pallets/ethereum-checked/src/tests.rs

```diff
@@ -21,14 +21,14 @@
 use super::*;
 use mock::*;
 
-use astar_primitives::ethereum_checked::MAX_ETHEREUM_TX_INPUT_SIZE;
+use astar_primitives::ethereum_checked::EthereumTxInput;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `astar_primitives::ethereum_checked::EthereumTxInput` is missing. Please add the following import statement:

```diff
+use astar_primitives::ethereum_checked::EthereumTxInput;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->